appium/tests/specs/mock/setup/CheckRefreshKeyFromEkmWithPassPhrasePrompt.spec.ts

```diff
@@ -59,19 +59,7 @@ describe('SETUP: ', () => {
       await KeysScreen.openScreenFromSideMenu();
       await KeysScreen.checkKeysScreen([ekmKeySamples.key0, ekmKeySamples.key1]);
 
-      // stage 4 - modified key gets updated, removed key does not get removed
```

Comment:
@tomholub I removed this test because this test doesn't seem to be valid now.

Reply:
You should instead change it, to test for the expectation that the key will get removed

Reply:
I already implemented it in https://github.com/FlowCrypt/flowcrypt-ios/blob/86d9c10f64129f3bd292aa3e8b0dd23448b7e4d7/appium/tests/specs/mock/setup/CheckRefreshKeyFromEkmKeyDeletionAndRevokedKey.spec.ts#L54 :)